app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -305,9 +305,18 @@ protected void onDestroy() {
         super.onDestroy();
         // unregister our receiver
         this.unregisterReceiver(this.mReceiver);
+        KissApplication.getCameraHandler().releaseCamera();
     }
+    
+    
 
     @Override
+	protected void onPause() {		
+		super.onPause();
+		KissApplication.getCameraHandler().releaseCamera();
+	}
+
+	@Override
```

Comment:
Sorry @Neamar, I'm using eclipse + adt plugin and by default it use real tabs instead of spaces (like your code seems to use). Probably the best thing is to write some guide for coding style (and better share editor settings from your IDE). Thanks :)